src/domains/queries.ml

```diff
@@ -37,7 +37,32 @@ end
 module LS = SetDomain.ToppedSet (Lval.CilLval) (struct let topname = "All" end)
 module TS = SetDomain.ToppedSet (CilType.Typ) (struct let topname = "All" end)
 module ES = SetDomain.Reverse (SetDomain.ToppedSet (CilType.Exp) (struct let topname = "All" end))
-module LiftedExp = Lattice.Flat(CilType.Exp)(struct let top_name = "Top" let bot_name = "Unreachable" end)
+module LiftedExp =
+struct
+  include Lattice.Flat(CilType.Exp)(struct let top_name = "Top" let bot_name = "Unreachable" end)
+  let to_invariant = function
+    | `Lifted e -> Some e
+    | `Bot | `Top -> None
+  let of_invariant = function
+    | Some e -> `Lifted e
+    | None -> `Top (* TODO: ??? *)
```

Comment:
Why the `???`? When would this need to be called?

Reply:
As I realized in https://github.com/goblint/analyzer/pull/759#discussion_r898791795, this is nonsense because `None` is used for both tautologies and dead paths. This will disappear when I do the refactoring.